### ISSUE TITLE
Platting  carrier code error

### ISSUE THREAD

**Issue by m88120** [Travelport/travelport-uapi-tutorial-c-sharp#82]
Hi,
please hlep me 
Host error during ticket issue. UNAUTHORISED/INVALID PLATING CARRIER CODE.

Thanks

**Comment by vivekjyotipramanik** [comment]
Hi  m88120,

Could you please confirm if you are using Trial Credentials. if yes, the ticketing will not work as ticketing requires extra provisioning and virtual printer to be set up. 

If you are using PreProduction credentials, Please check with your Travelport Account Manager and he will help you with the provisioning. Please let us know how it goes. Thanks.

**Comment by m88120** [comment]
yes I an using trial crediential. procedure to get the requires extra provisioning and virtual printer to be set up

**Comment by vivekjyotipramanik** [comment]
Hi m88120,

To get PreProduction credentials, you have to sign a contract with Travelport. Please contact with Travelport sales team and they will walk you through the procedure. Thanks.

**Comment by m88120** [comment]
i am not able to create or book a ticket from trial crendiential right?

**Comment by vivekjyotipramanik** [comment]
Hi  m88120,

Using trial credentials, it is possible to book a reservation, but you would not be able to book a ticket as ticketing requires a virtual printer which must be setup. Thanks.

**Comment by m88120** [comment]
how I can set up the virtual printer. please let me know in breif

**Comment by vivekjyotipramanik** [comment]
Hi m88120,

Only your Travelport Account Manager can help with setup the printer. Please contact your Travelport Account Manager regarding this. Thanks.

**Comment by m88120** [comment]
if I book ticket it will return me universallocater in response it mean ticket reservation is sucessfully done right?

**Comment by vivekjyotipramanik** [comment]
Hi  m88120,

The UniversalRecordLocator code is being returned in AirCreatereservationReq, that is not ticketing. Through AirCreateReservationReq you are holding the AirSegment. But using AirTicketingReq, you have to confirm the itinerary and you will get the Ticket Number in AirTicketRsp. In AirCreateReservationRsp you will get the UniversalRecordLocator code and that means you have successfully held the itinerary for further processing. Please let me know if this clarifies your question. Thanks.

**Comment by m88120** [comment]
Thanks if i need any things else I will get back to you. I will create ticket to travel port support regarding virtual printer sert up

thanks

**Comment by m88120** [comment]
Hi 
Please let me know in search passenger list BookingTravelerRef is dynamically generated by me or it return from API.  I can pass dynamic value in this.
Thanks

**Comment by vivekjyotipramanik** [comment]
Hi m88120,

The bookingTravelerRef in the LowFareSearch request and the AirPrice Request, you can provide any Alpha numeric numbers and UAPI will auto generate the reference in the response. We can use the auto generated reference key from the response, in the subsequent requests. Please find below the link with the details about the Keys in UAPI. Please let us know how it goes. Thanks.

https://support.travelport.com/webhelp/uapi/uAPI.htm#Getting_Started/Sending_Requests/DurableKeys.htm%3FTocPath%3DGetting%2520Started%7CSending%2520Requests%7C_____5
